src/Plugin/Condition/PathAliasExists.php

```diff
@@ -73,11 +74,19 @@ public static function create(ContainerInterface $container, array $configuratio
   }
 
   /**
-   * {@inheritdoc}
+   * Check if a path alias exists.
+   *
+   * @param string $alias
+   *   The alias to see if exists.
+   * @param mixed $language
+   *   The language to use.
+   *
+   * @return bool
+   *   TRUE if the system path does not match the given alias (ie: the alias
+   *   exists).
    */
-  public function evaluate() {
-    $alias = $this->getContextValue('alias');
-    $language = $this->getContext('language')->hasContextValue() ? $this->getContextValue('language')->getId() : NULL;
+  public function doEvaluate($alias, LanguageInterface $language = NULL) {
+    $language = !is_null($language) ? $language->getId() : NULL;
```

Comment:
no need for this line we can just pass on $language, the alias manager also accepts NULL.